website/docs/docs/cloud/about-cloud/dark-mode.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "dbt Cloud UI dark mode"
+id: dark-mode
+description: "About dbt Cloud UI dark mode"
+sidebar_label: dbt Cloud UI dark mode
+---
+
+dbt Cloud supports **Light mode** (default), **Dark mode**, and **System mode** (automatic theme switching based on your browser’s preferences set) under the **Theme** section of your user profile. You can seamlessly switch between these modes directly from the profile menu, customizing your viewing experience.
+
+Your selected theme is stored in your user profile, ensuring a consistent experience across dbt Cloud.
+
+Theme selection applies across all areas of dbt Cloud, including the [IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), [dbt Explorer](/docs/collaborate/explore-projects), [environments](/docs/environments-in-dbt), [jobs](/docs/deploy/jobs), and more. Learn more about customizing themes in [Enable dark mode in the dbt Cloud UI](add link)
+
+## Prerequisites
+
+- You have a dbt Cloud account. If you don’t, try [dbt Cloud for free!](https://www.getdbt.com/signup)
+- Access to dbt Cloud.
```

Comment:
What do we mean by "access" here? How is it different from having a cloud account and can they be combined?

SHould we say which plan you need? Does it work for developer, enterprise and team accounts?